tests/testsuite/nodejs.rs

```diff
@@ -4,11 +4,28 @@ use std::io;
 
 use crate::common;
 
+fn render_node_module() -> std::process::Command {
+    let mut command = common::render_module("nodejs");
+    if cfg!(windows) {
```

Comment:
Could we get a comment here explaining why `SYSTEMROOT` is needed in Windows?

Reply:
Done!